### PR TITLE
DDCNLS-6515: Updated to new Auth

### PR DESCRIPTION
**Pull request by matthew-mcnally**
Needs CYSP-Acceptance test merged for DDCNLS-6515 first.

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html [hunk 1]
```diff
@@ -16,32 +16,29 @@
...
-    @main(MainTemplateParams(
-        Some(Messages("nisp.main.title")),
-        userLoggedIn = true,
-        pageTitle = Some(Messages("nisp.excluded.title")),
-        gaDimensions = Some(analyticsAdditionalJs)
-    )) {
-        @exclusions.dead()
-    }
+@unauthenticatedMain(MainTemplateParams(
+    Some(Messages("nisp.main.title")),
```
[the remark this answers is not in the document]
Comment (BriWak) [reply app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html]: Done

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,21 +4,23 @@ object AppDependencies {
...
-    "uk.gov.hmrc" %% "csp-client" % "3.1.0",
-    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "0.2.0",
-    "uk.gov.hmrc" %% "local-template-renderer"  % "2.3.0",
-    "uk.gov.hmrc" %% "play-breadcrumb"  % "1.0.0"
+    "uk.gov.hmrc" %% "http-caching-client" % "8.5.0-play-25",
+    "uk.gov.hmrc" %% "play-language" % "4.2.0-play-25",
+    "uk.gov.hmrc" %% "tax-year" % "0.6.0",
+    "uk.gov.hmrc" %% "csp-client" % "4.1.0-play-25",
+    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.2.0-play-25",
+//    "uk.gov.hmrc" %% "local-template-renderer"  % "2.3.0",
```
[the remark this answers is not in the document]
Comment (BriWak) [reply project/AppDependencies.scala]: Done

File: app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html [hunk 3]
```diff
@@ -16,32 +16,29 @@
...
-@(exclusion: Exclusion, statePensionAge: Option[Int])(implicit messages: Messages, request: Request[_], user: NispUser,
+@import uk.gov.hmrc.nisp.controllers.auth.AuthenticatedRequest
+@(exclusion: Exclusion, statePensionAge: Option[Int])(implicit messages: Messages, request: Request[_],
   partialRetriever: CachedStaticHtmlPartialRetriever, application: Application, formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer)
 
 @analyticsAdditionalJs = @{
     Map[String, Any](
     "nispExclusionType" -> exclusion.toString,
-    "statePensionAge" -> statePensionAge,
-    "authProvider" -> user.authProvider,
```
[the remark this answers is not in the document]
Comment (BriWak) [reply app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html]: done

File: app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html [hunk 4]
```diff
@@ -25,92 +24,90 @@
...
-)(implicit messages: Messages, request: Request[_], user: NispUser, partialRetriever: CachedStaticHtmlPartialRetriever,
+)(implicit messages: Messages, request: Request[_], partialRetriever: CachedStaticHtmlPartialRetriever,
 application: Application,  formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer)
 
 
 @analyticsAdditionalJs = @{
     Map[String, Any](
         "nispExclusionType" -> exclusion.toString,
-        "statePensionAge" -> statePensionAge,
-        "authProvider" -> user.authProvider,
```
[the remark this answers is not in the document]
Comment (BriWak) [reply app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html]: Done

File: app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html [hunk 5]
```diff
@@ -14,22 +14,29 @@
...
+        templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer
+)
 
 @analyticsAdditionalJs = @{
     Map[String, Any](
     "nispExclusionType" -> exclusion.toString,
-    "statePensionAge" -> statePensionAge
+    "statePensionAge" -> statePensionAge,
+    "authProvider" -> request.authDetails.authProvider,
+    "authProvider" -> request.authDetails.confidenceLevel
```
Comment (daniel-manning) [inline app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html]: change to confidenceLevel

File: app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html [hunk 6]
```diff
@@ -15,35 +15,37 @@
...
 application: Application,  formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer)
 
-
 @analyticsAdditionalJs = @{
     Map[String, Any](
         "nispExclusionType" -> exclusion.toString,
-        "statePensionAge" -> statePensionAge
+        "statePensionAge" -> statePensionAge,
+        "authProvider" -> request.authDetails.authProvider,
+        "authProvider" -> request.authDetails.confidenceLevel
```
Comment (daniel-manning) [inline app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html]: Change to confidenceLevel

File: app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html (hunk 5 above)
Comment (jaynabonne) [reply app/uk/gov/hmrc/nisp/views/excluded_dead.scala.html]: Fixed.

File: app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html (hunk 6 above)
Comment (jaynabonne) [reply app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html]: Fixed.

**Comment by spattisson** [comment]
I've rerun these and they're all passing against this commit:
- [x] Acceptance tests
- [x] Zap pen/sec tests
- [x] Performance tests nisp-sapeak-performance-tests `nisp-run-local.sh` only

**Comment by daniel-manning** [comment]
:+1:

**Pull request merged**